docs/usage/python.md

```diff
@@ -89,8 +98,8 @@ The most direct way to disable all Python support in Renovate is like this:
 }
 ```
 
-Alternatively, maybe you only want one package manager, such as `npm`.
-In that case this would enable _only_ `npm`:
+Alternatively, you can use `enabledManagers` to tell Renovate what package managers it can use.
```

Comment:
I'm not sure we need a section on enabling specific managers, instead of disabling Python globally. It feels a bit out of place.

Reply:
Agreed, much of this document can probably be removed

Reply:
I want to remove this page later anyway, no sense to do too much work now